README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
Didn't even notice this. I'm pretty sure I already pointed it out a long time ago. This applies for only Windows and is not needed for compiling the source code.